charts/super-agent/templates/install-job.yaml

```diff
@@ -22,11 +22,22 @@ spec:
       containers:
         - name: apply-crs
           image: bitnami/kubectl
-          command:
-            - kubectl
-            - apply
-            - -f
-            - /manifests/
+          command: [ "/bin/bash","-c" ]
+          args:
```

Comment:
I just did a quick check on this with using a bersion like `version: ">=0.0.8-beta"` and the installation enters into an infinite loop of launching the job, woudl be possible to fix that state?

Reply:
`Jobs` in Kubernetes have a default `backoffLimit`  of 6, so it wouldn't happen to face an infinite loop here. However, let's set it explicitly in case the default change in the future and reduce it to 1.